packages/components/form-control/tests/form-control.test.tsx

```diff
@@ -59,73 +61,56 @@ describe("<FormControl />", () => {
         </FormControl>
       </>,
     )
-    expect(
-      screen
-        .getByTestId("FormControl-a")
-        .getElementsByClassName("ui-form__error-message")[0],
-    ).toHaveTextContent("Email is required.")
-    expect(
-      screen
-        .getByTestId("FormControl-b")
-        .getElementsByClassName("ui-form__helper-message")[0],
-    ).toHaveTextContent("Please enter your email")
+    expect(screen.queryByText("Email is required.")).toBeInTheDocument()
+    expect(screen.queryByText("Please enter your email")).toBeInTheDocument()
   })
 
   test("should be required", () => {
     render(
-      <FormControl data-testid="FormControl" label="Email" isRequired>
+      <FormControl label="Email" isRequired>
         <Input type="email" />
       </FormControl>,
     )
-    expect(
-      screen.getByTestId("FormControl").getElementsByTagName("input")[0],
-    ).toHaveAttribute("required")
+    expect(screen.getByRole("textbox")).toHaveAttribute("required")
```

Comment:
@illionillion 

You should use `.toBeRequired()` instead of `.toHaveAttribute("required")`.